blog/2024-02-16-announching-stryker-net-4.0.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: stryker4s-0-15-is-here
+title: 'Stryker.NET 4.0 is here!'
+authors: richard-werkman
+tags: [stryker-net]
+---
+
+Hello there! We're excited to share some fantastic news about the latest version of Stryker.NET. The team has been hard at work and has introduced some significant updates that will enhance the user experience and functionality of the platform.
+
+<!-- truncate -->
+
+## 💥 Breaking Change
+
+First and foremost, one of the most notable changes is the upgrade of the .NET runtime from dotnet 6 to dotnet 8. This update brings with it improved performance and LTS, ensuring that Stryker.NET remains at the forefront of development tools.
+
+## 🚀 What's New
+
+In addition to the runtime update, users will be thrilled to learn that the dashboard now supports live reporting. This means that users can now view and analyze real-time data, providing them with valuable insights into the performance of their projects. Even when Stryker is being run in a pipeline.
+
+## 📰 In Other News
+
+Since version `3.13.0` we support initializing a config file straight from the command line.
+
+```
+dotnet stryker init --project myproject.csproj
+```
+
+This command will create a new config file at the commands current location. It will fill the config file with default values for all file based options. Any options passed as CLI options will also be saved in the newly created config file. 
+
+## 🔮 What's Next?
+
+As part of the evolution of the platform, the decision has been made to deprecate the baseline & since options. They will be replaced with commands. 
```

Comment:
```suggestion
As part of the evolution of the platform, the decision has been made to deprecate baseline & since as global options. Instead they will be implemented as distinct subcommands to indicate that they are distinct modes of operation. 
```